### ISSUE TITLE
Command bc man page but no command

### ISSUE THREAD

**Issue by QuantRise** [QubesOS/qubes-issues#5083]
Qubes Version: R4.0

Problem: `man` page for `bc` command exists in dom0 but program doesn't. Inconsistent.

Proof: Do `whereis bc` in dom0, you will only see a reference to the `man` page. Can be verified by also going `man bc`

Solution:
    A) Add `bc` package because it's a standard GNU package that is usually included
    B) Remove `man` page for `bc` beacuse it doesn't make sense to have the `man` page installed without the program itself

Of course, attack surface of `dom0` should be limited but this program is very useful for calculating permission bit masks and again, it is a pretty standard tool in GNU. https://www.gnu.org/software/bc/bc.html

**Comment by andrewdavidwong** [comment]
I wonder whether the same bug exists in regular Fedora 25 the distro on which dom0 is based in Qubes 4.0).

**Comment by dmoerner** [comment]
I don't think this is a bug but just the way Fedora (and really originally Red Hat) does things, the package `man-pages` contains this man page and many others for important programs. You'll notice it's located in section 1p, signifying that it's supposed to reflect the POSIX standards for this program. I don't think that has to entail that an implementation of that standard is installed on the computer.

There are many more examples of POSIX manpages in `man-pages` for packages that aren't installed in dom0, e.g., `c99`.

You could remove `man-pages`, but I think many of the man pages are very useful to have (e.g., `xattr(7)`, etc.)
